WTWSMS/decisions/VIET_religion_decisions.txt

```diff
@@ -987,12 +997,22 @@ decisions = {
 		only_playable = yes
 		potential = {
 			has_landed_title = k_israel
-			religion_group = jewish_group
+			OR = {
+				religion_group = jewish_group
+				religion = ebionites
+			}
 			NOT = { trait = house_of_david }
 			NOT = { has_dynasty_flag = claimed_descent_from_david }
 		}
 		allow = {
-            piety = 200
+		            piety = 200
+			hidden_tooltip = { # Character must be ethnically Hebrew (AKA graphical culture AKA physical appearence, not culture) as David, or
+#					# passable by ancient norms to be Hewbrew (since no DNA tests exist, Sephardis could LARP as Hebrews if they were lucky)
+				text = MUSTHEBREW
+
+				race = hebrew
+				race = sephardi
+			}
```

Comment:
Race is CULTURE_GFX.

Whenever a character is born, his race is determined at random from his two parents. So if a Hebrew and an Indian man have a kid, he could be ethnically Hebrew or ethnically Indian, though his portrait will be something like MUSLIMGFX, etc. .

The Avatar Mod does NOT have a special race system. Everything it uses to determine race is from code that could be used in any Vanilla setting. I know, because I work on that Mod's editing board.